src/event.ts

```diff
@@ -197,16 +188,15 @@ export interface DuplicateDependency extends ErrorBase<ScriptReference> {
 export interface Cycle extends ErrorBase<ScriptReference> {
```

Comment:
I'm pushing that back for at least a little bit longer, because I think it may be helpful to give more info when dependencies don't resolve. e.g. to give one error when a dependency doesn't resolve because there is no package.json file there, and another error when the script isn't there